.circleci/config.yml

```diff
@@ -1,44 +1,214 @@
-version: 2
-jobs:
-  build: # runs not using Workflows must have a `build` job as entry point
-    resource_class: medium+
+version: 2.1
+orbs:
+  build-tools: circleci/build-tools@2.7.0
+executors:
+  tests_executor:
     docker: # run the steps with Docker
-      - image: circleci/openjdk:11.0.4-jdk-stretch
+      - image: cimg/openjdk:11.0.8
         environment:
           PGHOST: 127.0.0.1
-      - image: circleci/postgres:9.6.5-alpine-ram
+      - image: circleci/postgres:11.8
+        command: postgres -c max_connections=200
         environment:
           POSTGRES_USER: postgres
           POSTGRES_DB: postgres
+          PG_HOST: localhost
+          POSTGRES_HOST_AUTH_METHOD: trust
+          POSTGRES_PASSWORD: postgres
+      - image: docker.elastic.co/elasticsearch/elasticsearch:7.10.0
+        environment:
+          - xpack.security.enabled: false
+          - transport.host: localhost
+          - network.host: 127.0.0.1
+          - http.port: 9200
+          - discovery.type: single-node
+    resource_class: medium+
```

Comment:
If you don't need Postgres/ElasticSearch, see previous comment, then this probably doesn't need to be a medium+

Reply:
Removed the resource_class